Rakefile

```diff
@@ -302,8 +324,33 @@ We use information from your github account throughout the class.
         verify: -> { assert(!github_username.to_s.empty?, "We can't find your github username.", "") }
       },
 
+      global_gitignore: {
+        header: "Register a global gitignore file",
+        installation_steps: [
+          %q(
+1. Backup your existing global_gitignore (if it exists).  You can ignore a "No such file or directory" error:
```

Comment:
Attempting to be a good citizen and support developers who have an existing gitignore_global file.  This way, they can merge the files after InstallFest.